diff --git a/refind/main.c b/refind/main.c
index 35ecc4c..0d545fe 100644
--- a/refind/main.c
+++ b/refind/main.c
@@ -222,6 +222,9 @@ static VOID AboutrEFInd(VOID)
 #elif defined(EFIAARCH64)
         AddMenuInfoLine(&AboutMenu, PoolPrint(L" Platform: ARM (64 bit); Secure Boot %s",
                                               secure_mode() ? L"active" : L"inactive"));
+#elif defined(EFIARM)
+        AddMenuInfoLine(&AboutMenu, PoolPrint(L" Platform: ARM (32 bit); Secure Boot %s",
+                                              secure_mode() ? L"active" : L"inactive"));
 #else
         AddMenuInfoLine(&AboutMenu, L" Platform: unknown");
 #endif
